x-pack/plugins/observability/server/plugin.ts

```diff
@@ -14,23 +20,76 @@ import {
   AnnotationsAPI,
 } from './lib/annotations/bootstrap_annotations';
 import type { RuleRegistryPluginSetupContract } from '../../rule_registry/server';
+import { PluginSetupContract as FeaturesSetup } from '../../features/server';
 import { uiSettings } from './ui_settings';
 import { registerRoutes } from './routes/register_routes';
 import { getGlobalObservabilityServerRouteRepository } from './routes/get_global_observability_server_route_repository';
 
 export type ObservabilityPluginSetup = ReturnType<ObservabilityPlugin['setup']>;
 
+interface PluginSetup {
+  features: FeaturesSetup;
+  ruleRegistry: RuleRegistryPluginSetupContract;
+}
+
+const OBS_CASES_ID = 'observabilityCases';
+const OBSERVABILITY = 'observability';
+
 export class ObservabilityPlugin implements Plugin<ObservabilityPluginSetup> {
   constructor(private readonly initContext: PluginInitializerContext) {
     this.initContext = initContext;
   }
 
-  public setup(
-    core: CoreSetup,
-    plugins: {
-      ruleRegistry: RuleRegistryPluginSetupContract;
-    }
-  ) {
+  public setup(core: CoreSetup, plugins: PluginSetup) {
+    plugins.features.registerKibanaFeature({
+      id: OBS_CASES_ID,
+      name: i18n.translate('xpack.observability.featureRegistry.linkObservabilityTitle', {
+        defaultMessage: 'Cases',
+      }),
+      order: 1100,
+      category: DEFAULT_APP_CATEGORIES.observability,
+      app: [OBS_CASES_ID, 'kibana'],
+      catalogue: [OBSERVABILITY],
+      cases: [OBSERVABILITY],
+      management: {
+        insightsAndAlerting: ['triggersActions'],
```

Comment:
Sorry, my understanding of cases is very limited. I'm just confirming that you intend to show the alerting stack management page for users that are granted this feature. Is that correct?

<img width="336" alt="image" src="https://user-images.githubusercontent.com/3493255/121203253-f7c24480-c843-11eb-950a-e99d995922dc.png">

Reply:
I don't think we want that haha, that was a copy/paste issue on my part. @stephmilovic we can remove the whole `management` sections of the registration, or whatever is needed to get the type checker to succeed.